SETUP.md

```diff
@@ -71,69 +71,63 @@ Finally, flick the repository's switch to enable CI.
 Generate a deploy key so Travis CI can write to the repository.
 
 ```sh
+# IMPORTANT: change working directory to ci
 cd ci
+
+# Generate deploy.key.pub (public) and deploy.key (private)
 ssh-keygen \
-  -t rsa \
-  -b 4096 \
-  -C "travis@travis-ci.com" \
-  -N "" \
+  -t rsa -b 4096 -N "" \
+  -C "deploy@travis-ci.com" \
   -f deploy.key
 
-# For convenience, print the URL to add the public key to GitHub
-echo https://github.com/$OWNER/$REPO/settings/keys
+# Encode deploy.key to remove newlines, writing encoded text to deploy.key.txt.
+# This is required for entry into the Travis settings.
+openssl base64 -A -in deploy.key > deploy.key.txt
 ```
 
-Manually add the text of `deploy.key.pub` (with write access) to GitHub under the repository's deploy key settings (the URL echoed above).
-Give the key a descriptive title, such as "Travis CI Manubot".
-
-For the next step, you need the [Travis command line client](https://github.com/travis-ci/travis.rb) installed.
-This program is a Ruby gem:
-install it with `gem install travis` (not `apt install travis`, which is a different program).
-After the install, you will need to provide your credentials to login to travis with `travis login`.
+#### Add the public key to GitHub
 
 ```sh
-travis encrypt-file \
-  --com \
-  --repo=$OWNER/$REPO \
-  --force \
-  deploy.key > travis-encrypt-file.log
-```
+# Print the URL for adding the public key to GitHub
+echo "https://github.com/$OWNER/$REPO/settings/keys/new"
 
-`ci/travis-encrypt-file.log` will contain a line such as:
-
-```
-openssl aes-256-cbc -K $encrypted_8cc93e35fb85_key -iv $encrypted_8cc93e35fb85_iv -in test.key.enc -out test.key -d
+# Print the public key for copy-pasting to GitHub
+cat deploy.key.pub
 ```
 
-`ci/deploy.sh` must be updated with the random string from this line (i.e. `8cc93e35fb85`).
-The following command automates this substitution.
+Go to the GitHub settings URL echoed above in a browser, and click "Add deploy key".
+For "Title", add a description like "Manubot Travis Deploy Key".
+Copy-paste the contents of the `deploy.key.pub` text file (printed above by `cat`) into the "Key" text box.
+Check the "Allow write access" box below.
+Finally, click "Add key".
+
+#### Add the private key to Travis CI
 
 ```sh
-# Edit ci/deploy.sh with travis secure env variables generated by travis encrypt-file
-TRAVIS_ENCRYPT_ID=`python -c "import re; \
-  text = open('travis-encrypt-file.log').read(); \
-  print(re.search('encrypted_([a-f0-9]+)_key', text).group(1))"`
-sed "s/9befd6eddffe/$TRAVIS_ENCRYPT_ID/g" deploy.sh > tmp && mv -f tmp deploy.sh
+# Print the URL for adding the private key to Travis CI
+echo "https://travis-ci.com/$OWNER/$REPO/settings"
```

Comment:
You could add the following here to match the pattern above used for GitHub

```
# Print the encoded deploy.key for copy-pasting to Travis CI
cat deploy.key.txt
```

Reply:
Done in https://github.com/manubot/rootstock/pull/243/commits/2b030be93f37435e0a45c7c4b83f8af966c3b7c3 using:

```shell
cat deploy.key.txt && echo
```

`echo` (at least on linux) adds a newline so that the next command prompt is not printed on the same line as the output. Would be interested to have this tested on different OSes / shells.

Reply:
On Git Bash for Windows this creates a blank line between the end of the file contents and the next command prompt.  That isn't a problem.